utils/utils.py

```diff
@@ -78,6 +78,15 @@ def make_env(env_id, rank=0, seed=0, log_dir=None):
     def _init():
         set_global_seeds(seed + rank)
         env = gym.make(env_id)
+
+        # from baselines
+        #flatten_dict_observations = alg not in {'her'}
+        flatten_dict_observations = True
+        # check if it is a MiniGrid observation space
+        if flatten_dict_observations and isinstance(env.observation_space, gym.spaces.Dict):
```

Comment:
I think it would be better to add an helper method for flattening dict obs, as it will be re-used for creating a test env.

Reply:
Done.  In the same file for now, as flatten_dict_observations(env).
Feel free to tell your favor in code styles if any BTW.